packages/@react-stately/table/src/useTableColumnResizeState.ts

```diff
@@ -118,13 +124,15 @@ export function useTableColumnResizeState<T>(props: TableColumnResizeStateProps<
       columnLayout.getColumnMinWidth(key),
     getColumnMaxWidth: (key: Key) =>
       columnLayout.getColumnMaxWidth(key),
-    widths: columnWidths
+    widths: columnWidths,
+    tableState: state
```

Comment:
is it weird that i pass this through and join them here? or should people do the same thing to create this joined object that I did in TableView